### ISSUE TITLE
XCode 12: Failed to run in Simulator using xcframework (missing info.plist)

### ISSUE THREAD

**Issue by canius** [AppsFlyerSDK/AppsFlyerFramework#137]
# Report

## SDK Version
`5.3.0`

## What did you do?
use Mac Catalyst/AppsFlyerLib.xcframework in XCode 12 project

## What did you expect to happen?
can be run in simulator

## What happened instead?
cannot be running in simulator

## Please provide any other relevant information.
same issue for reference: [here](https://github.com/firebase/firebase-ios-sdk/issues/6360 )
![image](https://user-images.githubusercontent.com/420240/95814979-4f102380-0d4e-11eb-86ac-b3b8c16058bd.png)

**Comment by cutmail** [comment]
me too.

**Comment by rromanchuk** [comment]
lets gooo guys

**Comment by andr-ggn** [comment]
@rromanchuk 
Hello, can you try to reproduce an issue on empty project?

**Comment by conversun** [comment]
me too, I revert to cocoapods

**Comment by joshuapoq** [comment]
Any update on this issue @af-fess? 
Firebase suffered the same thing but were able to [promptly resolve with a hotfix](https://github.com/firebase/firebase-ios-sdk/pull/6444).
We have recently moved our platform to SPM and this is blocking our integration.

**Comment by joshuapoq** [comment]
Actually have a fix for this already; as it says just add a standard Info.plist into the `framework`s within the `xcframework` - I just made up a bundle identifier for them but I can create a PR if you need (as we need it)?
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>CFBundleExecutable</key>
	<string>AppsFlyerLib</string>
	<key>CFBundleIdentifier</key>
	<string>com.appsflyer.appsflyerlib</string>
	<key>CFBundleInfoDictionaryVersion</key>
	<string>6.0</string>
	<key>CFBundleName</key>
	<string>AppsFlyerLib</string>
	<key>CFBundlePackageType</key>
	<string>FMWK</string>
	<key>CFBundleVersion</key>
	<string>1</string>
	<key>DTSDKName</key>
	<string>iphonesimulator11.2</string>
</dict>
</plist>
```

**Comment by joshuapoq** [comment]
Thanks for the quick update guys! 🤗 This looks to be resolved in `6.1.0`!

**Comment by af-fess** [comment]
We added a plist: https://github.com/AppsFlyerSDK/AppsFlyerFramework/blob/master/MacCatalyst/AppsFlyerLib.xcframework/Info.plist
